src/applications/static-pages/tests/application-status.cypress.skip.spec.js

```diff
@@ -0,0 +1,114 @@
+import Timeouts from 'platform/testing/e2e/timeouts';
+
+const VA_FORM_IDS = require('platform/forms/constants').VA_FORM_IDS;
+
+Cypress.Commands.add('testStatus', (page, url) => {
+  cy.visit(page);
+  cy.get('.sip-application-status').should('be.visible', {
+    timeout: Timeouts.slow,
+  });
+  cy.injectAxeThenAxeCheck();
+
+  cy.get('main a.usa-button-primary').should('have.attr', 'href', url);
+
+  cy.get('.usa-button-secondary', { timeout: Timeouts.normal })
+    .should('exist')
+    .then(button => {
+      cy.wrap(button).click();
+    });
+  cy.get('#start-over-modal-title', { timeout: Timeouts.normal }).should(
+    'contain',
+    'Starting over will delete your in-progress application.',
+  );
+  cy.axeCheck();
+});
+
+describe('Application Status Test', () => {
```

Comment:
Do y'all care about counting skipped tests in your data? If so, could use `describe.skip` here.

Reply:
it's not going to be permanent. Honestly, this wound up in here by mistake. It's for a future batch of tests that I'm currently working on converting from nightwatch.  I figured I'd let it skip for now, and then when it's done it'll end up counted properly.

Reply:
removed it from the branch on this PR